tests/driver_epd_bw_spi_disp_dev/Makefile

```
@@ -0,0 +1,17 @@
+include ../Makefile.tests_common
+
+BOARD := nucleo-f411re
```

```suggestion
BOARD ?= nucleo-f411re
```

To allow using other boards :wink: